hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -11651,7 +11651,16 @@
       pinout:
       {
         remove_pads: []
-        add_pads: []
+        add_pads:
+        [
+          {
+            name: AST_MISC
+            type: InputStd
```

Comment:
ah i completely forgot, if it's input only the pad input is probably hard wired...